syntax = "proto3";
package cmdb_approve;




import "google/protobuf/struct.proto";

/*
GetHistoryObjectList请求
*/
message GetHistoryObjectListRequest {
    /*
    查询条件、类似mongodb
    */
    google.protobuf.Struct query = 1 ;
}

/*
GetHistoryObjectList返回
*/
message GetHistoryObjectListResponse {
    /*
    模型id列表
    */
    repeated string objectIdList = 1 ;
}

/*
GetHistoryObjectListApi返回
*/
message GetHistoryObjectListResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    cmdb_approve.GetHistoryObjectListResponse data = 4 ;
}

